hm-module.nix

```diff
@@ -42,12 +42,12 @@ in
 
   config = mkIf cfg.enable {
 
-    home.packages = [ finalPackage cfg.krewPackage ];
+    home.packages = [ finalPackage cfg.krewPackage pkgs.git ];
```

Comment:
I think you should use `home.extraActivationPath = [ pkgs.git ];` instead so you don't need to expose your `$PATH` to the script.

Reply:
It is an internal module option so you need to look at the source code here: https://github.com/nix-community/home-manager/blob/be47a2bdf278c57c2d05e747a13ed31cef54a037/modules/home-environment.nix#L434